doc/tools/docgram/common.edit_mlg

```diff
@@ -2144,7 +2142,7 @@ SPLICE: [
 | tac2def_val
 | tac2def_typ
 | tac2def_ext
-| tac2def_syn
+| ltac2def_syn
```

Comment:
The trick was that you needed to replace the token being spliced here, removing it wasn't sufficient.